contrib/docker/Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM debian:bookworm-slim
 
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
```

Comment:
Are these required?

Reply:
I was under the impression that they were required. That doesn't seems to be the case. I've removed them.